discretize/Tests.py

```diff
@@ -356,8 +364,8 @@ def l2norm(x):
         print(sadness[np.random.randint(len(sadness))]+'\n')
 
 
+    @requires({'matplotlib': matplotlib})
```

Comment:
I had no idea you could `@wrap` if statements. Cool.

Reply:
hmm, this seems to error on me. Is there something I need to have installed or updated? (That is what is causing the travis fail too...) 

Maybe we just put these lines of code in a function?